src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.util.*;
+
+public class UtilsTest {
+
+	@Test
+	public void testIsNull() {
```

Comment:
ote that we have a special naming convention for test method names. Please refer coding standard for more details.